backend/dissemination/search.py

```diff
@@ -17,7 +17,10 @@ def search_general(
     # TODO: use something like auditee_name__contains
     # SELECT * WHERE auditee_name LIKE '%SomeString%'
     if names:
-        names_match = Q(Q(auditee_name__in=names) | Q(auditor_firm_name__in=names))
+        names_match = Q()
+        for name in names:
+            names_match.add(Q(auditee_name__search=name), Q.OR)
```

Comment:
do we downcase/normalize anywhere?

Reply:
not explicitly, because this (`__search`) uses [psql's full text search](https://www.postgresql.org/docs/9.4/textsearch-intro.html), which is case-insensitive by default as far as I can tell (and it does in fact behave as such in our tests)